fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -1693,8 +1694,13 @@ public void logAlterRepository(Repository repo) {
         logEdit(OperationType.OP_ALTER_REPOSITORY, repo);
     }
 
-    public void logRestoreJob(RestoreJob job) {
```

Comment:
Doris' editlog has a replay process, and the specific steps are in EditLog.loadJournal. Both the log phase and the replay phase need to add the binlog to the binlog manager. If a RestoreBinLogInfo is temporarily created in the log phase, then it will not be possible to construct RestoreBinLogInfo in the replay phase.